src/Abp/Application/Features/FeatureCheckerExtensions.cs

```diff
@@ -11,12 +11,12 @@ namespace Abp.Application.Features
     /// Some extension methods for <see cref="IFeatureChecker"/>.
     /// </summary>
     public static class FeatureCheckerExtensions
-    {       
+    {
         /// <summary>
-        /// Gets value of a feature by it's name. This is sync version of <see cref="IFeatureChecker.GetValueAsync(string)"/>
+        /// Gets the value of a feature by its name. This is the sync version of <see cref="IFeatureChecker.GetValueAsync(string)"/>
         /// 
-        /// This is a shortcut for <see cref="GetValue(IFeatureChecker, int, string)"/> that uses <see cref="IAbpSession.TenantId"/> as tenantId.
-        /// So, this method should be used only if TenantId can be obtained from the session.
+        /// This is a shortcut for <see cref="GetValue(IFeatureChecker, int, string)"/> that uses <see cref="IAbpSession.TenantId"/>.
+        /// Note: This method should only be used if the TenantId can be obtained from the session.
```

Comment:
Hmm, "only if" is a common way of speech.

- [https://www.google.com.sg/search?q="be+used+only+if"](https://www.google.com.sg/search?q=%22be+used+only+if%22) (25,200,000 results)
- [https://www.google.com.sg/search?q="only+be+used+if"](https://www.google.com.sg/search?q=%22only+be+used+if%22) (16,800,000 results)

Reply:
Agreed

Reply:
There are multiple instances of this.

Reply:
There are still multiple instances of this.

Reply:
Thanks, @acjh